resources/dicts/relationship_events/normal_interactions/comfortable/increase.json

```diff
@@ -136,7 +149,9 @@
 		"id": "comfortable_kit_inc_med2",
 		"interactions": [
 			"m_c wants to snuggle with r_c.",
-			"m_c is hiding under a bush from r_c, but {PRONOUN/m_c/subject} can't stop giggling."
+			"m_c is hiding under a bush from r_c, but {PRONOUN/m_c/subject} can't stop giggling.",
+			"m_c wonders aloud if r_c would play with {PRONOUN/m_c/object} within earshot of r_c glancing over at {PRONOUN/m_c/object} expectantly.",
```

Comment:
This is rather unclear, see if you can reword it.  Perhaps something like: "m_c notices r_c glancing over at them expectantly. They pretend not to notice and instead wonder loudly if r_c would want to play with them."

Reply:
Hope this wording is better